ui/imports/shared/popups/addaccount/panels/WatchOnlyAddressSection.qml

```diff
@@ -24,10 +26,30 @@ Column {
         id: d
 
         property bool incorrectChecksum: false
+        readonly property string uuid: Utils.uuid()
```

Comment:
we want to create a new one for each request so should be assigned when the call is made?